README.md

```diff
@@ -2,36 +2,47 @@
 
 # Zammad admin documentation
 
-If you want to contribute to zammad documentation you can
-edit the \*.rst files and create pull requests.
+Source files for [Zammad’s admin documentation][docs].
 
-We take care about the translation part, so please don't change anything else
-whithin the repo, those changes will be discarded anyway ;) .
+## Contributing
 
-**Note:**
-These docs are available only in English.
-There are currently no plans to produce official translations into other languages.
+If you would like to improve the docs, simply:
 
-## Documentation
+1. fork the repo,
+2. edit the appropriate `.rst` files
+   (see [Markup Format](#restructuredtext-markup) below), and
+3. submit a pull request.
 
-Zammad hosts a searchable version of this documentation on https://docs.zammad.org
+> 🌍 **Wanna help translate?** Submit your contributions
+> (or request additional languages) [here][wbetranslate].
+> Do **NOT** submit a PR with changes to the contents of the `locale/` directory.
+
+Thanks! ❤ ❤ ❤  
+   The Zammad Team
 
 ### ReStructuredText markup
 
-If you like to edit the docs use the ReStructuredText markup language. Infos about this markup language can be found at:
+If you like to edit the docs use the ReStructuredText markup language.
+Information about this markup language can be found at:
 
 - http://www.sphinx-doc.org/en/stable/rest.html
 - http://docs.readthedocs.io/en/latest/_themes/sphinx_rtd_theme/demo_docs/source/demo.html
 
-Thanks! ❤️ ❤️ ❤️
+### Versioning
+
+This documentation provides versions:
 
-  Zammad Team
+- ``pre-release`` can contain develop, not yet releases functions and changes
```

Comment:
not yet `released`

Reply:
good call. Will fix that in both versions.